src/credentials/vue/ServiceItem.vue

```diff
@@ -21,9 +25,19 @@
             </div>
         </div>
     </button>
+
+    <!-- 
+        The form of the service item that will appear when the user clicks on it.
+        
+        Note: We only want to show this version of the form under the service item
+              when the user is in 'portrait' mode, otherwise we render it beside
+              the service item elsewhere in the code.
+    -->
+    <ServiceItemForm v-if="isSelected && !isLandscape" class="column-item"></ServiceItemForm>
```

Comment:
This seems awkward? I mean it works though it'd be better to use CSS selectors to change the container element

Reply:
I found out about vue `slots` and was able to move this code in to `root.vue` it should be less awkward